pkg/volume/cephfs/cephfs_test.go

```diff
@@ -27,8 +29,13 @@ import (
 )
 
 func TestCanSupport(t *testing.T) {
+	tmpDir, err := ioutil.TempDir("/tmp", "fake")
```

Comment:
I recommend using [```os.TempDir()```](https://golang.org/pkg/os/#TempDir) instead of the string ```/tmp``` in the case ```/tmp``` isn't the local temporary directory.